xradar/accessors.py

```diff
@@ -165,6 +173,13 @@ def get_crs(self):
         radar = self.xarray_obj
         return radar.pipe(get_crs)
 
+    def apply(self, func, *args, **kwargs):
```

Comment:
Nice, for DataArray and Dataset this looks like a reimplementation of `.pipe`? The thing is, that we can't use `DataTree.pipe` for our use case. So, yes, let's work with this as suggested.

Reply:
Okay, got it.